app/controllers/accounts/connected_accounts/selected_email_controller.rb

```diff
@@ -14,6 +14,7 @@ def edit
         @select_email_form = build_select_email_form
         @can_add_email = EmailPolicy.new(current_user).can_add_email?
         analytics.sp_select_email_visited
+        @email_id = @identity.email_address_id || last_email
```

Comment:
Can we add test coverage for both scenarios of when identity's existing `email_address_id` is assigned and `last_email` is assigned?

We have some existing coverage here for other variables, to build from:

https://github.com/18F/identity-idp/blob/3523bf5f301bdb2dc92abd01aae53a050cc6f862/spec/controllers/accounts/connected_accounts/selected_email_controller_spec.rb#L26-L32

Reply:
Added coverage in [1f40213](https://github.com/18F/identity-idp/pull/11568/commits/1f40213997a2fa7ee84cacac3579bae23ada5681)